test/test.js

```diff
@@ -12,6 +12,79 @@ const Type = UT.Anonymous;
 const Named = UT.Named;
 const Class = UT.Class;
 
+test.only('static sum.case supports multiple types', t => {
```

Comment:
Thank you!  I usually avoid `.only` for that exact reason, but in this case the code coverage would have caught it, which runs every `npm test`